### ISSUE TITLE
Pacman:6 doesn't honor multiple CacheDir in parralele download mode.

### ISSUE THREAD

**Issue by bartoszek** [AladW/aurutils#817]
Seems like new parallel download mode introduced in `pacman:6` breaks local repo caching.
Despite having `CacheDir=/home/_pacman_cache/aur` in `/etc/pacman.conf`
`pacman -Suy` still _downloads_ packages from local repo to the main pacman cache
 if `ParrallelDownload` is enabled in `/etc/pacman.conf`.

Should I report it upstream, or would you like to have the honor ☺️

**Comment by AladW** [comment]
I'm mostly surprised it took 5 years before this trick stopped working. 
Feel free to report it upstream.

**Comment by AladW** [comment]
@bartoszek can you test if symlinking packages in /var/cache/pacman/pkg 
alleviates the issue?

**Comment by bartoszek** [comment]
Nope, symlink is deleted and replaced with "downloaded" package file.

**Comment by AladW** [comment]
...how about a hardlink then?

Thanks for filing the report.

**Comment by Earnestly** [comment]
And then bind mounting.

**Comment by bartoszek** [comment]
@AladW setting `SigLever = Never` in the local repository section solves the issue [#substentiation](https://bugs.archlinux.org/task/71109#comment200263)

**Comment by AladW** [comment]
I'll update the man page to either suggest `SigLevel = Never` or `SigLevel = Always`. Having some packages signed in a local repo and others not (`SigLevel = Optional`) never made much sense anyway.

**Comment by abouvier** [comment]
`SigLevel = Always` does not exist, it's `SigLevel = Required` ;-)
